sarracen/render.py

```diff
@@ -762,13 +766,17 @@ def arrowplot(data: 'SarracenDataFrame',  # noqa: F821
     if data.get_dim() == 2:
         if not len(target) == 2:
             raise ValueError('Target vector is not 2-dimensional.')
+        if dens_weight is None:
+            dens_weight = False
         img = interpolate_2d_vec(data, target[0], target[1], x, y, kernel,
                                  x_arrows, y_arrows, xlim, ylim, exact,
                                  backend, dens_weight, normalize, hmin)
     elif data.get_dim() == 3:
         if not len(target) == 3:
             raise ValueError('Target vector is not 3-dimensional.')
         if xsec is not None:
+            if dens_weight is None:
+                dens_weight = False
```

Comment:
Also in the if block with `interpolate_3d_vec()`, first check if dens_weight is None and set to True if so.

Basically, dens_weight should be False for 2D data and 3D cross sections (slices), but True for 3D column-integrated views (which is `interpolate_3d_vec`).